src/test/incremental/hashes/closure_expressions.rs

```diff
@@ -95,7 +95,7 @@ pub fn add_type_ascription_to_parameter() {
 }
 
 #[cfg(not(cfail1))]
-#[rustc_clean(cfg="cfail2", except="HirBody")]
+#[rustc_clean(cfg="cfail2", except="HirBody, MirValidated, TypeckTables")]
```

Comment:
@michaelwoerister is this a reasonable change? I have no understanding of what is going here.

Reply:
From a conversation with Niko: "probably"

Reply:
Hard too tell. But if it's the only change required, then it can't be that bad.